'''

Wir haben nun gesehen wie wir (mehrere) Fehler abfangen können. 
Allerdings müsste man öfters nachdem die Fehlerbehandlung durchgeführt wurde
auch abschließende Aktionen durchführen.

Wie können wir so etwas durchführen? Lass uns einen Fall durchgehen!

'''


def main():
    pass


if __name__ == "__main__":
    main()